### PR TITLE
Add missing crcmod and nrf52 dependencies

### LINKED ISSUE #992: CRCmod module missing from Coretools dependencies
In tox tests for another package, I started getting this recently:
```
----------------------------- Captured stderr call -----------------------------
WARNING:iotile.core.dev.registry:Unable to load sensorgraph from Distribution('iotile_support_lib_controller_3', '3.8.5')
Traceback (most recent call last):
  File "/var/app/.tox/py/lib/python3.6/site-packages/iotile/core/dev/registry.py", line 167, in load_extensions
    ext = entry.load()
  File "/var/app/.tox/py/lib/python3.6/site-packages/entrypoints.py", line 82, in load
    mod = import_module(self.module_name)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/app/.tox/py/lib/python3.6/site-packages/iotile_support_lib_controller_3/sensorgraph.py", line 7, in <module>
    from iotile.core.utilities.hash_algorithms import KNOWN_HASH_ALGORITHMS
  File "/var/app/.tox/py/lib/python3.6/site-packages/iotile/core/utilities/hash_algorithms/__init__.py", line 4, in <module>
    from .crc import calculate_crc
  File "/var/app/.tox/py/lib/python3.6/site-packages/iotile/core/utilities/hash_algorithms/crc.py", line 1, in <module>
    import crcmod
ModuleNotFoundError: No module named 'crcmod'
------------------------------ Captured log call -------------------------------
WARNING  iotile.core.dev.registry:registry.py:169 Unable to load sensorgraph from Distribution('iotile_support_lib_controller_3', '3.8.5')
Traceback (most recent call last):
  File "/var/app/.tox/py/lib/python3.6/site-packages/iotile/core/dev/registry.py", line 167, in load_extensions
    ext = entry.load()
  File "/var/app/.tox/py/lib/python3.6/site-packages/entrypoints.py", line 82, in load
    mod = import_module(self.module_name)
```

It seems that Coretools now has a dependency on the `crcmod` package, but it's not listed in it's dependencies, which caused the tox failure.

I'm not sure which version of `crcmod` it should be pegged to. @anwu Do you know?

### PR DESCRIPTION
**Pull request by timothyfroehlich**
## Overview & Major Changes

Fixes #992 
This adds a missing dependency of iotile-core, crcmod.

While testing, I found that the tox config for iotile-core was missing the NRF52 support dependency, which caused the tests to fail when running tox in iotile-core instead of as part of the entire coretools package.

### CODE REVIEW

File: iotilecore/tox.ini [hunk 1]
```diff
@@ -8,4 +8,5 @@ deps=
     ../iotiletest
     ../iotileemulate
     pycryptodome
+    iotile-support-con-nrf52832-3
```
Comment (timburke) [inline iotilecore/tox.ini:11]: This dependency can't go here.  If there's been a code change that makes `iotile-core` need this installed before it works, then that code change needs to be reverted.

No open-source package can have a dependency on a closed source package.

Comment (timburke) [reply iotilecore/tox.ini:11]: Ah, sorry, for testing?  My mistake.  That would be fine.

Comment (timothyfroehlich) [reply iotilecore/tox.ini:11]: This is a dependency on a tox test. When I ran tox on the root directory of coretools I noticed many closed-source packages in the tox dependencies.

Comment (timothyfroehlich) [reply iotilecore/tox.ini:11]: Here's the list of dependencies that wer einstalled when I ran tox from the root directory:
```
py36-linux_only installdeps: six, cmdln, pytest, pytest-logging, pytest-localserver, ./iotilecore, ./iotilebuild, ./iotiletest, ./transport_plugins/socket_lib, ./transport_plugins/websocket, ./iotilegateway, ./iotilesensorgraph, ./iotileship, ./iotileemulate, ./transport_plugins/bled112, ./transport_plugins/awsiot, ./transport_plugins/jlink, iotile-support-lib-
controller-4, iotile-support-con-nrf52832-3, iotile-support-con-nrf52832-4, ./transport_plugins/native_ble, ./iotile_ext_cloud, requests-mock, tornado>=4.5.3,<5.0.0, pycryptodome, configparser
```

**Pull request merged**